src/pages/Calculator.js

```diff
@@ -42,8 +42,10 @@ const Calculator = () => {
 
   return (
     <div className='calculator'>
-      <CalculatorDisplay display={result} />
+      <h3>Let&apos;s do some math!</h3>
```

Comment:
- [ ] Please remove this unused [code](https://github.com/prius2055/Math-magicians/pull/5/files#diff-518ab039f8a31bc368b8b8e69ec8cdb3e1fa5849fcb27ea14c57ec2b3cefc946L2:~:text=//%20import%20propTypes%20from%20%27prop%2Dtypes%27%3B), and this unused [code](https://github.com/prius2055/Math-magicians/pull/5/files#diff-518ab039f8a31bc368b8b8e69ec8cdb3e1fa5849fcb27ea14c57ec2b3cefc946L2:~:text=//%20Calculator.propTypes%20%3D%20%7B,//%20%7D%3B) as well, thank you!

Reply:
I have cleaned the unused codes from Calculator and Navbar components

Reply:
In Calculator.js, line 70

```javascript
// Calculator.propTypes = {
//   onButtonClick: propTypes.func.isRequired,
// };
``` 

this one also removed, right?

Reply:
well resolved! ✔️

Reply:
@Rustamxon7 Thanks for that. Eagle eye!

Reply:
Hope it works for your benefit :)

Reply:
Spacibo Bratan!

Reply:
Pojalusta!